src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -101,7 +101,7 @@
               <small
                 >Don't see an organization?
                 <a
-                  href="https://github.com/settings/connections/applications/7ad54aa857c6503013ea"
+                  href="https://github.com/settings/connections/applications/{{ Dockstore.GITHUB_CLIENT_ID }}"
```

Comment:
Cool.  A bit out of scope, but if we're making this parameterizable, might as well also do it in the other places where it appears (grep for `7ad54aa857c6503013ea`)